joblib/memory.py

```diff
@@ -558,7 +558,7 @@ def call_and_shelve(self, *args, **kwargs):
         """Call wrapped function, cache result and return a reference.
 
         This method returns a reference to the cached result instead of the
-        result itself. The reference object is small and pickeable, allowing
+        result itself. The reference object is small and pickleable, allowing
```

Comment:
It seems that the term that is used in the [`pickle` module doc](https://docs.python.org/3/library/pickle.html#what-can-be-pickled-and-unpickled) is `picklable` so lets go with this? 
```suggestion
        result itself. The reference object is small and picklable, allowing
```